### PR TITLE
Create fake-api with Swagger documentation

### PR DESCRIPTION
**Pull request by arianerocha**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: '3.8'
```
Comment (rluders) [inline docker-compose.yml:1]: Can't we use `3.3`?

File: docker/docker-compose-swagger.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+version: '3.8'
```
Comment (rluders) [inline docker/docker-compose-swagger.yml:1]: Can't we use `3.3`?

File: fake-api/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+## Mocks and Docs
```
Comment (rluders) [inline fake-api/README.md:1]: Convert to `h1`

File: fake-api/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+All the REST API must use the OpenAPI Specification version 3 and the documentation will be centralized in this repository using swagger to build the mocks server.
+
+To mock server, we are using [openapi-mock](https://github.com/muonsoft/openapi-mock)
+
+---
```
Comment (rluders) [inline fake-api/README.md:7]: Do we really need this `hr`?

File: fake-api/README.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+## Table of Contents
+
+- [Swagger Files](#swagger-files)
+- [Commands](#commands)
+
+---
```
Comment (rluders) [inline fake-api/README.md:14]: Do we really need this `hr`?

**Comment by rluders** [comment]
@arianerocha in general it seems OK. Besides the comments that I added to the review, there is also two points that I would consider importants:

- I'm not against the usage of Makefile. But I was wondering if it would be possible to remove it and replace it by some npm scripts. Since the Makefile doesn't seems to be doing anything complicated and all the other commands of the project can be executed via npm/yarn.

- How to consume the fake API? I guess that this is a missing information.

**Comment by arianerocha** [comment]
> * How to consume the fake API

 * Section `How to consume the fake API` added on README;
 * About change the commands to npm script. I'm not totally against, I just feel a little bit strange to use npm scripts without be a node script.

**Comment by rluders** [comment]
> > * How to consume the fake API
> 
>  * Section `How to consume the fake API` added on README;
>  * About change the commands to npm script. I'm not totally against, I just feel a little bit strange to use npm scripts without be a node script.

We can keep the Makefile, no strong opinion here. It is working.

LGTM

**Review by rluders — APPROVED** [review]

**Pull request merged**